src/services/HttpClientService.js

```diff
@@ -0,0 +1,24 @@
+import axios from "@/axios.config";
+
+async function fetchData(endpoint) {
+    try {
+        const response = await axios.get(endpoint);
+        if (response.status === 200) {
+            return response.data;
+        }
+    } catch (error) {
+        throw error;
+    }
+}
+
+export async function fetchResources() {
+    return await fetchData("/resources");
+}
+
+export async function fetchProducts() {
+    return await fetchData("/products");
+}
+
+export async function fetchSystemEvents() {
+    return await fetchData("/systemEvents");
+}
```

Comment:
We don't have a systemEvents endpoint yet. But we have users. Can you remove the fetchSystemEvents method, add a method for the users endpoint and remove the UsersService as the code for fetching data from an endpoint is duplicated now.